src/start-express-app.ts

```diff
@@ -30,20 +30,40 @@ export default async function(options: N9NodeRouting.Options): Promise<N9NodeRou
 		controllers: [options.path + '/**/*.controller.*s'],
 	};
 
+	// Configure morgan
+	morgan.token('total-response-time', (req: Request, res: Response, digits: number): string => {
+		if (!(req as any)._startAt) {
```

Comment:
@petitcl I suppose that `_startAt` is set by Morgan ?

Reply:
yep